src/app/core/services/repo-creator.service.ts

```diff
@@ -18,9 +20,43 @@ export const BUG_REPORTING_INVALID_ROLE =
 export class RepoCreatorService {
   constructor(
     private githubService: GithubService,
-    private userService: UserService
+    private userService: UserService,
+    private repoCreationConfirmationDialog: MatDialog
   ) {}
 
+  /**
+   * Checks if the current repository has been created and if not, checks
```

Comment:
Let's make the comment more straightforward.
`Prompts user to allow CATcher to create a repo, if the repo does not exist and the current phase is the bug reporting phase`

Reply:
Alright, we can do so! :+1: